src/components/transformation/create/GitPodButton.tsx

```diff
@@ -108,16 +161,24 @@ function GitPodButton({
                     );
                 }
 
-                const [token, draftId] = await Promise.all([
+                const [token, evaluatedDraftId] = await Promise.all([
                     createRefreshToken(false, '1 day'),
                     generateDraftWithSpecs(),
                 ]);
 
+                if (!evaluatedDraftId) {
+                    throw new Error(
+                        intl.formatMessage({
+                            id: 'newTransform.errors.urlNotGenerated',
```

Comment:
Since we refrain from exposing the concept of a draft to the user, I felt this error message could be reused in this scenario:

`We failed to generate the proper URL to start GitPod. Try again and if the issue persists please contact support.`